utils/src/main/scala/za/co/absa/enceladus/utils/types/Format.scala

```diff
@@ -61,7 +67,15 @@ object Format {
   def epochFactor(format: String): Long = {
     format.toLowerCase match {
       case "epoch"      => 1
-      case "milliepoch" => 1000
+      case "milliepoch" => thousand
```

Comment:
If you are trying to "bypass" magic numbers here, why not change the upper as well?